.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[REPORT]"
+labels: kind/bug-report, needs-triage
+assignees: ''
+
+---
+
+**Describe the bug**
+_A clear and concise description of what the bug is._
+
+**To Reproduce**
+_Steps to reproduce the behavior:_
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+_A clear and concise description of what you expected to happen._
+
```

Comment:
We should also have an actual behavior section.  What behavior should we have seen, and what behavior did we actually see?

Reply:
The one I ended up pushing is the one suggested by GitHub.

I had the same concern, but then thought that for how the section is structured the actual behavior is detailed in the steps to reproduce.
Also I thought that a lot of time, I saw the actual behavior section being a repetition of the steps.

I was waiting for a discussion like this one to decide whether or not to have the "actual behavior" section. I'd propose not to for now, and add it in future if we feel we need it.

Based on your experience, what do you think? Would it be enough just having the "steps to reproduce" block?

Reply:
With this context, I'm fine with it either way, though I'd like to hear some other opinions.